docs/sources/alerting/set-up/provision-alerting-resources/_index.md

```diff
@@ -14,128 +14,68 @@ labels:
     - cloud
     - enterprise
     - oss
-title: Import and export Grafana Alerting resources
+title: Provision Alerting resources
 weight: 300
 ---
 
-# Import and export Grafana Alerting resources
+# Provision Alerting resources
 
 Alerting infrastructure is often complex, with many pieces of the pipeline that often live in different places. Scaling this across multiple teams and organizations is an especially challenging task. Importing and exporting (or provisioning) your alerting resources in Grafana Alerting makes this process easier by enabling you to create, manage, and maintain your alerting data in a way that best suits your organization.
 
 You can import alert rules, contact points, notification policies, mute timings, and templates.
 
 You cannot edit imported alerting resources in the Grafana UI in the same way as alerting resources that were not imported. You can only edit imported contact points, notification policies, templates, and mute timings in the source where they were created. For example, if you manage your alerting resources using files from disk, you cannot edit the data in Terraform or from within Grafana.
 
-To modify imported alert rules, you can use the **Modify export** feature to edit and then export.
+## Import alerting resources
+
+Choose from the options below to import (or provision) your Grafana Alerting resources.
 
-Choose from the options below to import your Grafana Alerting resources.
+1. [Use configuration files to provision your alerting resources][alerting_file_provisioning], such as alert rules and contact points, through files on disk.
 
-1. Use file provisioning to manage your Grafana Alerting resources, such as alert rules and contact points, through files on disk.
+   **Note:**
```

Comment:
you can remove this Note - the admonition short code type note takes care of all that for you:)

Reply:
fixed by https://github.com/grafana/grafana/pull/82221/commits/abbd39ca219b9e8d137ae46320a4fc0ee7c15456